MeshLib/Mesh.h

```diff
@@ -125,6 +125,16 @@ class Mesh : BaseLib::Counter<Mesh>
     MeshLib::Properties & getProperties() { return _properties; }
     MeshLib::Properties const& getProperties() const { return _properties; }
 
+    bool isAxiallySymmetric() const { return _is_axially_symmetric; }
+    void setAxiallySymmetric(bool is_axial_symmetric) {
```

Comment:
It may be needed for some benchmarking, and it can be considered in future if it is really needed.